cmd/policy/policy.go

```diff
@@ -364,3 +378,35 @@ func prettyJSONEncoder(dst io.Writer) *json.Encoder {
 	enc.SetIndent("", "  ")
 	return enc
 }
+
+//getPolicyDecisionLocally takes path of policy path/directory and input (eg build config) as string, and performs policy evaluation locally
+func getPolicyDecisionLocally(policyPath, inputString string) (*cpa.Decision, error) {
+	var input interface{}
+	if err := yaml.Unmarshal([]byte(inputString), &input); err != nil {
+		return nil, fmt.Errorf("invalid input: %w", err)
+	}
+
+	var parsedPolicy *cpa.Policy
+	pathInfo, err := os.Stat(policyPath)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get path info: %w", err)
+	}
+
+	//if policyPath is directory get content of all files in this directory (non-recursively) and add to document bundle
+	if pathInfo.IsDir() {
+		parsedPolicy, err = cpa.LoadPolicyDirectory(policyPath)
+	} else {
+		parsedPolicy, err = cpa.LoadPolicyFile(policyPath)
+	}
+	if err != nil {
+		return nil, fmt.Errorf("failed to load policy files: %w", err)
+	}
+
+	ctx := context.Background()
```

Comment:
Small fix here. We require clear, helpful variable names to be used.

Reply:
Is this comment for `ctx`? Do you have a recommendation for this? I believe this is standard name for context instances.
Eg. https://github.com/kubernetes/kubernetes/blob/902e507824068fe1f6cb7044a6b599dc3c285b7a/staging/src/k8s.io/apiserver/pkg/storage/etcd3/compact_test.go#L31